README.md

```diff
@@ -1,4 +1,10 @@
 # coverage-gutters
+
+![Version](https://img.shields.io/visual-studio-marketplace/v/ryanluker.vscode-coverage-gutters)
+![Installs](https://img.shields.io/visual-studio-marketplace/i/ryanluker.vscode-coverage-gutters)
+![Downloads](https://img.shields.io/visual-studio-marketplace/d/ryanluker.vscode-coverage-gutters)
+![Rating](https://img.shields.io/visual-studio-marketplace/r/ryanluker.vscode-coverage-gutters)
```

Comment:
Just a suggestion, happy to change to whatever makes the most sense.

Reply:
These look great thanks @mattseddon should help folks when comparing the different coverage displaying extensions 😉.